docs/api/navigators/TabNavigator.md

```diff
@@ -85,6 +85,7 @@ The route configs object is a mapping from route name to a route config, which t
 - `tabBarPosition` - position of the tab bar, can be `'top'` or `'bottom'`
 - `swipeEnabled` - whether to allow swiping between tabs
 - `animationEnabled` - whether to animate when changing tabs
+- `configureTransition` - a function that returns a configuration object that describes the animation between tabs
```

Comment:
Your flag helped me realize that we don't need to pass `configureTransition` to `TabBarComponent` at all. It seems like [the code does that for `animationEnabled`](https://github.com/react-community/react-navigation/blob/v1.0.0-beta.12/src/views/TabView/TabView.js#L115) and I did the same 🙈